### PR TITLE
DB: Deprecate - _get_json_meta

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/db.php [hunk 1]
```diff
@@ -115,6 +116,8 @@ public function get_builder( $post_id, $status = self::STATUS_PUBLISH ) {
 	 * @return array Decoded JSON data from post meta.
 	 */
 	protected function _get_json_meta( $post_id, $key ) {
+		_deprecated_function( __METHOD__, '3.1.0', 'get_post_meta( $post_id, $key, true ) and then json_decode( $meta, true )' );
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply includes/db.php:119]: We don't use it anywhere. I just deprecate it for 3rd party usage.

[the remark this answers is not in the document]
Comment (kobizz) [reply includes/db.php:119]: It has already done [here](https://github.com/elementor/elementor/pull/12434/files#diff-8e563e348f0815e931a2e282aa993008f6eec0fc19b679d907c424deca3e16a9)

**Comment by Nevoss** [comment]
Already executed by: https://github.com/elementor/elementor/pull/12434

**Comment by Nevoss** [comment]
Already executed by: #12434

**Pull request closed without merging**